vexim/config/header.php

```diff
@@ -13,9 +15,9 @@
     print     "-- " . $_SESSION['domain'] . " ";
   }
   if (isset($_SESSION['domain_id'])) {
-    if (($domheaderrow['enabled'] == "0") || ($domheaderrow['enabled'] == "f")) {
+    if (($domheaderrow['enabled'] === "0") || ($domheaderrow['enabled'] === "f")) {
```

Comment:
Was this put for a new feature, or debugging? The `enabled`-field is either `1` or `0`